src/Symfony/Bridge/Monolog/Tests/Processor/SwitchUserTokenProcessorTest.php

```diff
@@ -26,8 +28,13 @@ class SwitchUserTokenProcessorTest extends TestCase
 {
     public function testProcessor()
     {
-        $originalToken = new UsernamePasswordToken('original_user', 'password', 'provider', ['ROLE_SUPER_ADMIN']);
-        $switchUserToken = new SwitchUserToken('user', 'passsword', 'provider', ['ROLE_USER'], $originalToken);
+        if (class_exists(InMemoryUser::class)) {
+            $originalToken = new UsernamePasswordToken(new InMemoryUser('original_user', 'password', ['ROLE_SUPER_ADMIN']), 'provider', ['ROLE_SUPER_ADMIN']);
+            $switchUserToken = new SwitchUserToken(new InMemoryUser('user', 'passsword', ['ROLE_USER']), 'provider', ['ROLE_USER'], $originalToken);
+        } else {
+            $originalToken = new UsernamePasswordToken(new User('original_user', 'password', ['ROLE_SUPER_ADMIN']), null, 'provider', ['ROLE_SUPER_ADMIN']);
```

Comment:
Maybe make a comment here so it is more easily found when we remove code in Symfony 6.0. 

(or you maybe have added this code to your list of future PRs)